### DISCUSSION TITLE
Cacher des morceaux de codes

### DISCUSSION THREAD

**Discussion by MarlBnc** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#233] [Q&A] [answered]
Bonjour,

Dans le cadre de l'issue #223 , nous souhaitons implémenter un système de booster avec probabilité. Est-il possible de cacher les valeurs des proba afin qu'elle reste inconnue dans le code, ou sera t-il possible de vous demander de les enlever de l'OSS lors de la PR ? 

Merci

**Comment by NariieL** [comment, marked as answer]
Hello,

Il est inutile de cacher des probabilités.

À l'image de ZUnivers, tout est public et ça ne pose aucun souci.

**Reply by MarlBnc** [reply]
Merci de la réponse !
